scripts/install

```diff
@@ -190,7 +194,7 @@ ynh_script_progression --message="Starting a systemd service..."
 ###		- And the section "STOP SYSTEMD SERVICE" and "START SYSTEMD SERVICE" in the change_url script
 
 # Start a systemd service
-ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log"
+ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log" --line-match="[INFO] Available at http://:::9000/ in private mode"
```

Comment:
Randomly passing by and saw https://ci-apps-dev.yunohost.org/jenkins/job/thelounge_ynh%20(Rafi59)/lastBuild/console

Sounds like there's a typo in that option : 

```suggestion
ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log" --line_match="[INFO] Available at http://:::9000/ in private mode"
```